conda_windows.md

```diff
@@ -0,0 +1,88 @@
+---
+ Getting Going with Python on a Windows
+---
+
+
+#X
+
+The aim of this web page is to help you get started with Python on Windows. We will explain what a package management tool is, how to download `conda` package management tool via the Anaconda installer, and guide you on the Windows terminal so that you can use `conda` from the command line. Finally, we will wrap up by installing one library with `conda`.  
+
+##//What Is a Package Management Tool?
+
+A package management tool is a software application that helps you manage software libraries that enable you to get your work done. These software libraries may relate to plotting for scientific publication or accessing certain kinds of data, for example.
+
+When you start using Python, you may want to use software libraries that are not part of the standard Python installation. For example, you may wish to use the [Unidata metpy library](https://pypi.python.org/pypi/MetPy) for meteorological data and visualization. 
+
+Anaconda from Continuum Analytics will help you install `metpy` easily.
+
+##//Installing the `conda` Package Management Tool
+
+The `conda` package management tool is part of the Anaconda software package. Install `conda` by navigating to the [Anaconda download page](https://www.continuum.io/downloads). Scroll down to the “Anaconda for Windows” portion of the web page.
+
+Download the Python 3.5 version by clicking on the “Windows 64 bit Graphical Installer” link. Should we include a section on how to figure out if they want to install the 64 or 32 bit? It is a big download, so it is best to be on fast network. Open the installer file you just downloaded. It should be named something like: "Anaconda3-4.0.0-Windows-x86_64" 
```

Comment:
@ddirks any idea about this on windows? I'm not sure what the prevalence of 32-bit windows is these days.